runners/java-fn-execution/src/test/java/org/apache/beam/runners/fnexecution/control/SdkHarnessClientTest.java

```diff
@@ -573,7 +577,7 @@ public void handleCleanupWithStateWhenAwaitingOnClosingOutputReceivers() throws
   @Test
   public void verifyCacheTokensAreUsedInNewBundleRequest() throws InterruptedException {
     when(fnApiControlClient.handle(any(BeamFnApi.InstructionRequest.class)))
-        .thenReturn(createRegisterResponse());
+        .thenReturn(createAnyResponse());
```

Comment:
Since this function is only used once, you might want to inline the contents of the function here for readability.

Reply:
Makes sense. Done.